docs/build.sh

```diff
@@ -20,12 +20,5 @@
 
 set -e
 rm -rf build 2> /dev/null
-rm -rf temp 2> /dev/null
-mkdir temp
-cp -rf source/* temp/
-# replace relative URLs with absolute URLs
-sed -i -e 's/\.\.\/\.\.\/\.\.\//https:\/\/github.com\/apache\/arrow-datafusion\/blob\/main\//g' temp/contributor-guide/index.md
```

Comment:
We only ever did this to one file and it looks like the file doesn't have anymore relative URLs